index.bs

```diff
@@ -998,7 +998,7 @@ but short enough that the dangling promise will still be resolved in a reasonabl
 <pre class="idl">
     [SecureContext]
     partial interface PublicKeyCredential {
-        [Unscopable] Promise < boolean > isPlatformAuthenticatorAvailable();
```

Comment:
This [Unscopable] came from https://github.com/w3c/webauthn/pull/379#issuecomment-308037955, where I believe it was just a mistaken copy from a change to `Document`.